packages/contracts/test/minter-suite-minters/SEA/MinterSEA/views.test.ts

```diff
@@ -0,0 +1,159 @@
+import { expect } from "chai";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { ethers } from "hardhat";
+
+import { initializeProjectZeroTokenZeroAuctionAndSettle } from "./helpers";
+import { setupConfigWitMinterFilterV2Suite } from "../../../util/fixtures";
+import { deployAndGet, deployCore, safeAddProject } from "../../../util/common";
+import { ONE_MINUTE, ONE_HOUR, ONE_DAY } from "../../../util/constants";
+import { Logger } from "@ethersproject/logger";
+// hide nuisance logs about event overloading
+Logger.setLogLevel(Logger.levels.ERROR);
+
+const TARGET_MINTER_NAME = "MinterSEAV1";
+const TARGET_MINTER_VERSION = "v1.0.0";
+
+const runForEach = [
+  {
+    core: "GenArt721CoreV3",
+  },
+  // {
+  //   core: "GenArt721CoreV3_Explorations",
+  // },
+  // {
+  //   core: "GenArt721CoreV3_Engine",
+  // },
+  // {
+  //   core: "GenArt721CoreV3_Engine_Flex",
+  // },
```

Comment:
Is the commenting out of these intentional?

Reply:
it is for now - tests are still being written and doing this makes running them/iteration a bit quicker during development. They will be uncommented before tests are finalized.

Reply:
the tests are now ready and these are uncommented 👍